deploy/mainnet/700_depoy_vesper_RewardForwarder.ts

```diff
@@ -0,0 +1,20 @@
+import { DeployFunction } from "hardhat-deploy/types"
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts } = hre
+  const { deploy, get } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  await deploy("RewardForwarder", {
+    from: deployer,
+    log: true,
+    skipIfAlreadyDeployed: true,
+    args: [
+      (
+        await get("LiquidityGaugeV5_SaddleFRAXBPVesperFRAXMetaPoolLPToken")
+      ).address,
+    ],
+  })
```

Comment:
Since RewardForwarder wont be a unique deployment, we should add prefixes to the saved name.

Add `contract` key to the options with `RewardForwader` but save as something like `RewardForwarder_VSP_LiquidityGaugeV5_SaddleFRAXBPVesperFRAXMetaPoolLPToken`